xlive/GSCustomMenu.cpp

```diff
@@ -3763,8 +3764,19 @@ __declspec(naked) void sub_2111ab_CMLTD_nak_AccountList() {//__thiscall
 }
 
 void GSCustomMenuCall_AccountList();
-
+using namespace MapChecksumSync;
 static bool CMButtonHandler_AccountList(int button_id) {
+	// todo: better error here
+	if (MapChecksumSync::get_startup_info() != startup_state::done)
+	{
+		switch (MapChecksumSync::get_startup_info())
```

Comment:
why are we switching on a single option?

Reply:
I was planning to give decent error messages here but gave up.